java/test/src/main/java/io/ray/test/ActorRestartTest.java

```diff
@@ -31,6 +31,10 @@ public int increase() {
       return value;
     }
 
+    public boolean askRayWasCurrentActorRestarted() {
```

Comment:
what about `checkWasCurrentActorRestartedInActorTask`? and `wasCurrentActorRestarted` can be renamed to `checkWasCurrentActorRestartedInActorCreationTask`